docs/ApplyConfig.md

```diff
@@ -71,6 +71,22 @@ If you would rather use the filevault-package-maven-plugin for building the pack
 
 The `*.yaml` files are installed directly from the package content and respect the [run mode semantics](Configuration.md). Otherwise there is no limitation, so the YAML files will be picked up from anywhere in the package (as long as the parent node does not contain a `.` followed by one or multiple not matching run modes).
 
+If you wish to limit which YAML files are picked up by the installation hook, you can add one or more regular expressions as package properties that start with the prefix `actool.hook.paths.patterns`. If the path of a file within the package matches any of the provided patterns, it will be picked up by the installation hook:
+
+```
+<plugin>
+    <groupId>org.apache.jackrabbit</groupId>
+    <artifactId>filevault-package-maven-plugin</artifactId>
+    <configuration>
+        <properties>
+            <installhook.actool.class>biz.netcentric.cq.tools.actool.installhook.AcToolInstallHook</installhook.actool.class>
+            <actool.hook.paths.patterns.groups>/jcr_content/apps/myapp/acl/groups.*</actool.hook.paths.patterns.groups>
+            <actool.hook.paths.patterns.users>/jcr_content/apps/myapp/acl/users.*</actool.hook.paths.patterns.users>
```

Comment:
Looking at this example:
*  I would prefer using the same prefix for AC tool package properties (arguably the existing one isn't perfect, but to stay consistent at least)
* I think regex allows to nicely cater for multiple paths (we don't expect long lists here), e.g. like  `/jcr_content/apps/myapp/acl/(groups|users)/.*`
* I would allow to omit the path prefix `/jcr_content` (or maybe even only support without it, `jcr_content` in package is an implementation detail)

The above would make look the example like this:
```
<installhook.actool.class>biz.netcentric.cq.tools.actool.installhook.AcToolInstallHook</installhook.actool.class>
<installhook.actool.configFileRegex>/apps/myapp/acl.*</installhook.actool.configFileRegex>
```

@santiagozky WDYT?

Reply:
thanks @ghenzler.
I'll do the following:
-Again prepend the /jcr_root since it's an implementation detail. (so not optional, let's not introduce more complexity into this)
-Rename the property to _installhook.actool.configFilesPattern_ (or anything _installhook.actool.configFilesPattern .*_ to support multiple ones as @kwin suggested )
-fix documentation

Reply:
errata: the installhook prefix cannot be used. I guess it is reserved for setting up a new hook
>Request failed: org.apache.jackrabbit.vault.packaging.PackageException: Invalid installhook property: installhook.actool.configFilesPattern